src/esp/physics/PhysicsManager.h

```diff
@@ -96,7 +96,7 @@ class PhysicsManager : public std::enable_shared_from_this<PhysicsManager> {
   PhysicsManager. Each entry suggests a derived class of @ref PhysicsManager and
   @ref RigidObject implementing the specific interface to a simulation library.
   */
-  enum PhysicsSimulationLibrary {
+  enum class PhysicsSimulationLibrary {
```

Comment:
Would now be a good time to refactor these enumerators to be Pascal case?

Reply:
Fun migration. Need to work around the `None` keyword in python. Refactoring to `NoPhysics` instead.

Reply:
BTW, I'm leaving the config and metadata tag "none" for now.